src/templates/Plugin.js

```diff
@@ -82,13 +78,23 @@ const PluginTemplate = ({ data }) => {
 
       <SitewideHeader />
 
-      <Header plugin={plugin} />
-
-      <main className="pt-4 pb-8 px-4 lg:pb-28">
-        <div className="relative max-w-lg mx-auto lg:max-w-4xl">
-          <Body plugin={plugin} siteMetadata={siteMetadata} />
-          <Notes plugin={plugin} />
-          <PluginCTA plugin={plugin} />
+      <div className="mt-4">
+        <Header plugin={plugin} />
+      </div>
+
+      <main className="pb-8 px-4 lg:pb-28">
+        <div className="relative max-w-7xl mx-auto">
+          <div className="grid grid-cols-3 gap-20">
+            <article className="col-span-2">
+              <Body plugin={plugin} siteMetadata={siteMetadata} />
```

Comment:
Next improvement will probably split all this content into tabs to make the pages easier to read.